src/main/java/com/vinylteam/vinyl/util/impl/DetailedVinylParserImpl.java

```diff
@@ -16,7 +16,7 @@
 @RequiredArgsConstructor
 public class DetailedVinylParserImpl implements DetailedVinylParser {
 
-    private final ParserConfiguration config;
+    final ParserConfiguration config;
 
```

Comment:
why changed access modifier?

Reply:
to make it visible from descendants, but seems protected is more applicable